tools/deploy_ipk.sh

```diff
@@ -57,6 +65,10 @@ main() {
                 CERTIFICATION_MODE=true
                 shift
                 ;;
+            --proxy)
+                SSH_OPTIONS="$SSH_OPTIONS \"-oProxyJump $2\""
```

Comment:
`$2` should also be quoted.

Reply:
All these quotes are giving me a headache :)

Reply:
Still brought them back.